### PR TITLE
fix: Handle ticket fee not found error in send monthly invoice

### LINKED ISSUE #6869: Error while sending monthly invoice
```
NoResultFound: No row was found for one()
  File "app/api/helpers/scheduled_jobs.py", line 256, in send_monthly_event_invoice
    db.session.query(TicketFees).filter_by(currency=currency).one()
  File "sqlalchemy/orm/query.py", line 3354, in one
    raise orm_exc.NoResultFound("No row was found for one()")
ObjectNotFound: ({'source': ''}, 'Ticket Fee not set for None')
  File "app/api/helpers/scheduled_jobs.py", line 260, in send_monthly_event_invoice
    {'source': ''}, 'Ticket Fee not set for {}'.format(currency)
```

We should never be throwing unhandled exceptions in jobs where no one can handle them

```
events = [
<Event 'Yogesh Sharma'>
, 
<Event '"/><img src="x" onerror="alert(document.domain)">'>
, 
<Event 'testing'>
, 
<Event 'Hackathon'>
, 
<Event 'Jamia Hamdard Convocation'>
, 
<Event 'OpenTechSummit Day Mekong'>
, 
<Event 'FOSSASIA Meetup - PSLab'>
, 
<Event 'Introduction to open source'>
, 
<Event 'Yogesh Sharma'>
, 
<Event 'OpenTechSummit India'>
]

event = <Event 'Yogesh Sharma'>
```

### PR DESCRIPTION
**Pull request by codedsun**
Fixes #6869 

#### Short description of what this resolves:
 Ticket fee not found error in send monthly invoice

#### Changes proposed in this pull request:
Skip that entry and log an error

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The unit tests pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

- [x] All the functions created/modified in this PR contain relevant docstrings.

### CODE REVIEW

**Comment by codedsun** [comment]
@iamareebjamal - Am I logging the error in a wrong way?

File: app/api/helpers/scheduled_jobs.py [hunk 1]
```diff
@@ -256,9 +256,9 @@ def send_monthly_event_invoice():
                     db.session.query(TicketFees).filter_by(currency=currency).one()
                 )
             except NoResultFound:
-                raise ObjectNotFound(
-                    {'source': ''}, 'Ticket Fee not set for {}'.format(currency)
-                )
+                app.logger.error('Ticket Fee not found for event id {id}'.format(id=event.id))
```
Comment (iamareebjamal) [inline app/api/helpers/scheduled_jobs.py:259]: Don't use app.logger

Create a new logger with `__name__`

**Comment by codedsun** [comment]
@iamareebjamal - Review

File: app/api/helpers/scheduled_jobs.py [hunk 2]
```diff
@@ -256,9 +256,10 @@ def send_monthly_event_invoice():
                     db.session.query(TicketFees).filter_by(currency=currency).one()
                 )
             except NoResultFound:
-                raise ObjectNotFound(
-                    {'source': ''}, 'Ticket Fee not set for {}'.format(currency)
-                )
+                logging.getLogger(__name__)\
```
Comment (iamareebjamal) [inline app/api/helpers/scheduled_jobs.py:259]: No

Comment (codedsun) [reply app/api/helpers/scheduled_jobs.py:259]: @iamareebjamal - Shall I create a global variable logger here? 

`logger = logging.getLogger(__name__)`

Comment (iamareebjamal) [reply app/api/helpers/scheduled_jobs.py:259]: Yes

**Pull request merged**